web/src/js/__tests__/ducks/settingsSpec.js

```diff
@@ -0,0 +1,23 @@
+jest.unmock('../../ducks/settings')
+jest.mock('../../utils')
+
+import reduceSettings from '../../ducks/settings'
+import * as SettingsActions from '../../ducks/settings'
```

Comment:
We can combine this to `import reduceSettings, * as SettingsActions from '../../ducks/settings'`